newIDE/app/src/AssetStore/ExtensionStore/ExtensionListItem.js

```diff
@@ -53,10 +53,13 @@ export const ExtensionListItem = ({
 
   // Report the height of the item once it's known.
   const containerRef = React.useRef<?HTMLDivElement>(null);
-  React.useLayoutEffect(() => {
-    if (containerRef.current)
-      onHeightComputed(containerRef.current.getBoundingClientRect().height);
-  });
+  React.useLayoutEffect(
+    () => {
+      if (containerRef.current)
+        onHeightComputed(containerRef.current.getBoundingClientRect().height);
+    },
+    [onHeightComputed]
```

Comment:
adding dependency array for good practice

Reply:
Same as before, let's check what this callback does to see if there is a risk this creates an infinite loop.